README.md

```diff
@@ -226,6 +226,60 @@ Transfer-Encoding: chunked
 }
 ```
 
+## Edit an user by using an existing user
+
+One user connected through the backend client is allowed to edit an user.
+
+```
+echo '
+{
+    "email": "TEST@EXAMPLE.COM" ,
+    "username" : "USER_NAME",
+    "phone_number" : "12345",
+    "roles": ["ROLE_1", "ROLE_2"]
+}
+' |  http POST http://127.0.0.1:8089/app_dev.php/admin/users/{id} 'Authorization:Bearer {accessToken}'
+```
+
+This API call throws an access denied exception if the user is not allowed to create another user.
+
+If everything is made correctly you should get back this
+
+```
+HTTP/1.1 200 OK
+Cache-Control: no-cache
+Connection: Keep-Alive
+Content-Type: application/json
+Date: XXX
+Server: XXXX
+Transfer-Encoding: chunked
+
+{
+    "id": 5
+}
+```
+
+## Disable an user by using an existing user
+
+One user connected through the backend client is allowed to disable other users.
+
+```
+http PATCH http://127.0.0.1:8089/app_dev.php/admin/users/{id}/disable 'Authorization:Bearer {accessToken}'
+```
+
+This API call throws an access denied exception if the user is not allowed to create another user.
```

Comment:
This sentence should be refactored in a common section for the backend API calls.